libs/ledgerjs/packages/devices/src/ble/receiveAPDU.ts

```diff
@@ -1,61 +1,90 @@
 import { TransportError, DisconnectedDevice } from "@ledgerhq/errors";
-import { Observable } from "rxjs";
-import { log } from "@ledgerhq/logs";
+import { Observable, ReplaySubject, takeUntil } from "rxjs";
+import { TraceContext, trace } from "@ledgerhq/logs";
 const TagId = 0x05;
-// operator that transform the input raw stream into one apdu response and finishes
-export const receiveAPDU = (rawStream: Observable<Buffer>): Observable<Buffer> =>
+
+/**
+ * Parses a raw stream coming from a BLE communication into an APDU response
+ *
+ * @param rawStream An observable containing the raw stream as emitted buffers
+ * @param options Optional options containing:
+ *   - context An optional context object for log/tracing strategy
+ * @returns An observable containing the APDU response as one emitted buffer
+ */
+export const receiveAPDU = (
```

Comment:
I think `parseBLERawStreamIntoAPDU` would be better yeah - It's just going to need changes in the 3 BLE transport (node and web too), and if any project uses it directly, it will be a breaking change. Maybe we should do this in another PR ?

Reply:
yep 👍